R/summarize_results.R

```diff
@@ -12,7 +12,11 @@ results_columns <- c(
   "histogram_end",
   "value",
   "metric",
-  "dist"
+  "dist",
+  "empirical_mean",
+  "empirical_var",
+  "empirical_sd",
+  "empirical_skew"
```

Comment:
I personally like "sample" more than empirical but your call 😁

Reply:
Your wish is my command!